### PR TITLE
[MIG] product_ux to 11.0

### PR DESCRIPTION
**Pull request by nicomacr**
Change module name "product_usability" to "product_ux"

### CODE REVIEW

File: product_ux/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#. Just install this module.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. No configuration nedeed.
+
+
```
Comment (zaoral) [inline product_ux/README.rst]: Missing Usage title.

File: product_ux/README.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=================
+Product Usability
```
Comment (zaoral) [inline product_ux/README.rst]: @jjscarafia The module display name will remain Product Usability or we want to updated to Product UX as well?

File: product_ux/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    ],
+    'depends': [
+        'product',
+    ],
+    'data': [
+        'views/product_template_views.xml',
+    ],
+    'demo': [
+    ],
+    'test': [
```
Comment (zaoral) [inline product_ux/__manifest__.py]: remove test key, this one is deprecated.

File: product_ux/i18n/es.po [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+"PO-Revision-Date: 2018-03-19 13:42+0000\n"
+"Last-Translator: Javier Diez <jd@adhoc.com.ar>, 2018\n"
+"Language-Team: Spanish (https://www.transifex.com/adhoc/teams/46451/es/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Language: es\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: product_usability
```
Comment (zaoral) [inline product_ux/i18n/es.po]: @jjscarafia, @nicomacr and I were talking about what to do with translation when we migrated. We do not have a steps to follow. Will be good to discuss in the next meeting

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @nicomacr 

Here are some technical review
Also I listed here some pylint that can be fixed

```bash
We need the next warnings in pylint, Please fix
************* Module product_ux
product_ux/__init__.py:1: [W8201(incoherent-interpreter-exec-perm), ] You have a python file with execution permissions but you don't have a interpreter magic comment. If you really needs a execution permission then add a magic comment ( https://en.wikipedia.org/wiki/Shebang_(Unix) ). If you don't needs a execution permission then remove it with: chmod -x __init__.py
product_ux/README.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
count_errors 0
```

I also maded functional review, everything working properly.

Best Regards

File: product_ux/README.rst (hunk 1 above)
Comment (nicomacr) [reply product_ux/README.rst]: Done!

File: product_ux/__manifest__.py (hunk 3 above)
Comment (nicomacr) [reply product_ux/__manifest__.py]: Done!

**Pull request merged**